tests/functional/aws-node-sdk/test/multipleBackend/objectTagging/objectTagging.js

```diff
@@ -274,7 +274,8 @@ function testSuite() {
                 });
             });
 
-            it('should return error on putting tags to object deleted from AWS',
+            it('should not return error on putting tags to object ' +
+            'that has had a delete marker put directly on from AWS',
```

Comment:
you can add tags to a delete marker? also, please clean up the grammar here a bit.

Reply:
I don't think you can. The test is:
1) put an object via s3 (version 'a' on AWS)
2) create delete marker directly on AWS
3) putting tags on object via s3 should put tags on version 'a' on AWS (we ignore the delete marker)